src/com/lilithsthrone/game/dialogue/utils/UtilText.java

```diff
@@ -652,7 +652,7 @@ public static String parse(List<GameCharacter> specialNPC, String input) {
 						
 						} else if (c == '(') {
 							if(command == null) {
-								command = sb.toString().substring(1); // Cut off the '.' at the start.
+								command = sb.toString().substring(1); // Cut off the '(' at the start.
```

Comment:
It is actually cutting off a '.' in the most common case, here. The '(' is the end marker of the substring.

Reply:
I thought it was a copypasta mistake